### PR TITLE
add episode 011 transcript

### PR DESCRIPTION
**Pull request by ericseppanen**
This was a difficult transcript; it has a larger than average number of
"unintelligible" placeholders.  Hopefully someone will correct those in
the future.

### CODE REVIEW

File: _transcripts/011-jake-yoshua-stjepan.md [hunk 1]
```diff
@@ -0,0 +1,1106 @@
...
+somehow? And then Yoshua told me to ask you about performance, in some way, and
+said that you working on performance, very heavily. Do you want to talk about
+that?
+
+__Stjepan__: So in `crossbeam`, we have our own channels which are based on
+`crossbeam` channels, in particular the bounded version of channels in
+`crossbeam`. So the performance advantage of `crossbeam` channels are in that,
+context switches on asynchronous tasks are much lower overhead than context
+switches with threads. So if you're sending a lot of data between different
+tasks, we can context switch between them much faster than with threads.
```
Comment (ericseppanen) [inline _transcripts/011-jake-yoshua-stjepan.md:1033]: I feel like there's something wrong with this part.  I don't know if it's me misunderstanding, or if the speaker just mis-spoke. At present the transcript effectively says "crossbeam channels are based on crossbeam channels", which doesn't make a lot of sense to me.

This paragraph starts at 50:29, if anyone wants to give it a listen and offer corrections.

**Comment by jonhoo** [comment]
This one is probably best if @bstrie takes a look at, since he's the one who did the interviews, and so heard them speak with his own ears!

**Comment by ericseppanen** [comment]
This is getting very stale.  Since a transcript with a few missing words is still more useful than no transcript at all, I think it would be best to merge this as-is.  If people find the time to work out what the missing words are, additional fixes can be applied later.

**Comment by jonhoo** [comment]
I agree. Merged!

**Pull request merged**